docs/cosmos-hub/README.md

```diff
@@ -1,22 +1,22 @@
-# Cosmos Hub Documentation
+# Cosmos-Hub Documentation
```

Comment:
Why `Cosmos-Hub` and not `Cosmos Hub`? We use `Cosmos Hub` in all our material. And it's two separate words, we link them together?

Reply:
oh okay I didn't realize, makes sense, just reversed